README.md

```diff
@@ -8,20 +8,36 @@ Once you’ve cloned this then `bundle` will install the requirements.
 
 ## Running the application
 
-Prepare the application configuration:
+The application makes use of environment variables for specifying certain configurable parameters.
+You can set these by creating a file named `.env` and defining any environment variables within. 
+The file `.env-example` lists all the required environment variables with example values - simply 
+copy it and change the values.
 
-```
+```bash
 cp .env-example .env
 vim .env
 ```
 
-You can run the application with:
+Once the required environment variables have been defined, you can run the application with:
 
-```
+```bash
 bundle exec rails server
 open localhost:3000
 ```
 
+#### Environment variables
```

Comment:
Nice one! But instead of adding it here (or as well as adding it here), I suggest adding as comments to the `env-example` file so it sits next to the code?